vita3k/mem/src/mem.cpp

```diff
@@ -142,6 +153,12 @@ Address alloc_at(MemState &state, Address address, size_t size, const char *name
 }
 
 void free(MemState &state, Address address) {
+    auto &addr_map = state.aligned_addr_to_original;
+    if (addr_map.find(address) != addr_map.end()) {
```

Comment:
Is it mandatory to do the find or you can just access with [old_addr]?

Reply:
It's mandatory since I didn't set the non aligned addr to the map.